snapshot/Program.cs

```diff
@@ -0,0 +1,133 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr.Tests.Snapshot
+{
+    using Dapr.Actors;
+    using Dapr.Actors.Client;
+    using Dapr.Client;
+    using Dapr.Tests.HashTagApp.Actors;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading.Tasks;
+
+    public class Program
+    {
+        // This uses the names of shapes for a generic theme
+        static internal string[] HashTags = new string[]
+        {
+            "circle",
+            "ellipse",
+            "square",
+            "rectangle",
+            "triangle",
+            "star",
+            "cardioid",
+            "picycloid",
+            "limocon",
+            "hypocycoid"
+        };
+
+        private static string[] Sentiments = new string[]
+        {
+            "verynegative",
+            "negative",
+            "neutral",
+            "strong",
+            "verystrong"
+        };
+
+        public static void Main(string[] args)
```

Comment:
This is not really a test. It is just a way to run these locally, right? Do we have unit tests?

Reply:
I'm not sure what this comment is pointing at, it looks like it's pointing at Main()?

It's not a test in itself, it's an app that's part of the pipeline.  There's no unit tests, every app really just does one thing.